scripts/zkai_attention_derived_d128_activation_swiglu_gate.py

```
@@ -0,0 +1,923 @@
+    cases: list[dict[str, Any]] = []
+    for name, mutator, refresh in MUTATION_BUILDERS:
+        mutated = copy.deepcopy(core_payload)
+        mutator(mutated)
+        if refresh:
+            refresh_payload_commitment(mutated)
+        try:
+            validate_payload(mutated, context=context)
+        except AttentionDerivedD128ActivationSwiGluError as err:
+            expected_error = EXPECTED_MUTATION_ERRORS.get(name)
+            if expected_error is None:
+                raise AttentionDerivedD128ActivationSwiGluError(f"mutation error marker missing: {name}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)